LoRaEngine/LoraKeysManagerFacade/CreateEdgeDevice.cs

```diff
@@ -99,10 +108,9 @@ static async Task<ConfigurationContent> GetConfigurationContentAsync(Uri configL
 
                 var deviceConfigurationContent = await GetConfigurationContentAsync(new Uri(Environment.GetEnvironmentVariable("DEVICE_CONFIG_LOCATION")), new Dictionary<string, string>
                 {
-                    ["[$region]"] = region,
                     ["[$reset_pin]"] = resetPin,
                     ["[$spi_speed]"] = string.IsNullOrEmpty(spiSpeed) || string.Equals(spiSpeed, "8", StringComparison.OrdinalIgnoreCase) ? string.Empty : ",'SPI_SPEED':{'value':'2'}",
-                    ["[$spi_dev]"] = string.IsNullOrEmpty(spiDev) || string.Equals(spiDev, "2", StringComparison.OrdinalIgnoreCase) ? string.Empty : ",'SPI_DEV':{'value':'1'}"
+                    ["[$spi_dev]"] = string.IsNullOrEmpty(spiDev) || string.Equals(spiDev, "0", StringComparison.OrdinalIgnoreCase) ? string.Empty : $",'SPI_DEV':{{'value':'{spiDev}'}}"
```

Comment:
Was the 2 a mistake here? I see it also as the default value in `azuredeploy.json`. In fact there we allow only 1 and 2 to be passed, should we amend this?